redact/package.json

```
@@ -0,0 +1,59 @@
+{
+  "name": "@arcjet/redact",
+  "version": "1.0.0-alpha.22",
+  "description": "Arcjet sensitive information redaction library.",
```

```suggestion
  "description": "Arcjet sensitive information redaction library",
```
Nit: I remove the periods here.